docs/releasing.md

```diff
@@ -0,0 +1,13 @@
+# Releasing uPortal-home
+
+## Apache release rules
+
+This project has [adopted Apache rules][] with necessary or pragmatic local adaptations.
```

Comment:
Whatever necessary or pragmatic local adaptations we need to make. :)

I think it's simultaneously true that

+ we're trying to become a more mature open source software project
+ we see in Apache some rules and culture and having-it-all-together-ness worth emulating
+ there's some interpretation needed in adapting this, because Apereo isn't Apache
+ figuring this out is messy.

Apache release rules formally would require a PMC vote, would require the release binaries being available to the PMC for review before public release, would require `+1` voters to actually verify those binaries, would require a bunch of vigor that we're not prepared to live up to at this point.  So this documentation is an attempt to acknowledge that reality.

Saying that one has a given policy or practice and then not actually having that policy or practice is poisonous, is erosive. Trying to avoid that particular sin and create some runway for figuring this all out.